app/views/categories/_categories.html.slim

```diff
@@ -0,0 +1,5 @@
+- for category in @categories
+  = category.amount
+  = category.title
+  = link_to t('edit'), edit_category_path(category),  {remote: true, 'data-toggle':  "modal", 'data-target': '#modal-window',  class: 'btn btn-outline-secondary'}
+  = link_to t('delete'), category_path(category), method: :delete, data: { confirm: t('are_you_sure') },  class: 'btn btn-outline-danger delete-tr'
```

Comment:
@marderer we can render a collection of entries via partial's `collection`. see: https://stackoverflow.com/a/1540846